libdevcore/Log.cpp

```diff
@@ -60,7 +60,7 @@ LogOutputStreamBase::LogOutputStreamBase(char const* _id, std::type_info const*
 	m_autospacing(_autospacing),
 	m_verbosity(_v)
 {
-	Guard l(x_logOverride);
+	x_logOverride.lock();
 	auto it = s_logOverride.find(_info);
 	if ((it != s_logOverride.end() && it->second) || (it == s_logOverride.end() && (int)_v <= g_logVerbosity))
```

Comment:
Do you know what this "log override" is about? Maybe it's not used / needed.

Reply:
It is an mutex with a name that has no relation to its function.

It is a static global used to synchronize use of non-thread safe stuff. I'd rename it to x_console_guard or something similar.